addons/dragon/data/M47.rvmat

```diff
@@ -0,0 +1,81 @@
+class StageTI {
+    texture="z\ace\addons\dragon\data\M47_TI_CA.tga";
+};
+ambient[]={0.958,0.98400003,1,1};
+diffuse[]={0.958,0.98400003,1,1};
+forcedDiffuse[]={0,0,0,0};
+emmisive[]={0,0,0,1};
+specular[]={0.153,0.169,0.111,1};
+specularPower=90;
+PixelShaderID="Super";
+VertexShaderID="Super";
+class Stage1 {
+    texture="z\ace\addons\dragon\data\M47_nohq.tga";
+    uvSource="tex";
+    class uvTransform {
+        aside[]={1,0,0};
+        up[]={0,1,0};
+        dir[]={0,0,1};
+        pos[]={0,0,0};
+    };
+};
+class Stage2 {
+    texture="a3\weapons_f\data\detailmaps\Metal_rough_DT.tga";
+    uvSource="tex";
+    class uvTransform {
+        aside[]={4,0,0};
```

Comment:
Are you sure about the 4 here?
Also these are direction vectors, they should add up to 1, not 4.